pi_cond.c

```diff
@@ -2,14 +2,25 @@
 // Copyright © 2018 VMware, Inc. All Rights Reserved.
 
 #include <stdio.h>
-#include "rtpi.h"
+#include "rtpi_internal.h"
 
 /*
  * This wrapper for early library validation only.
  * TODO: Replace with pthread_cond_t wrapper with a new cond implementation.
  *       Base this on the older version of the condvar, with the patch from
  *       Dinakar and Darren to enable priority fifo wakeup order.
  */
+
+pi_cond_t *pi_cond_alloc(void)
+{
+	return malloc(sizeof(pi_cond_t));
+}
+
+void pi_cond_free(pi_cond_t *cond)
+{
+	free(cond);
+}
+
 int pi_cond_init(pi_cond_t *cond, struct pi_mutex *mutex, uint32_t flags)
```

Comment:
We will need to add the new alloc/free functions to the README for completeness/consistency.